From f875857d5f5185d7c15e38c88dc28da2452be46f Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Sun, 30 Jul 2017 11:43:06 -0400
Subject: [PATCH 0713/4131] drm/amd/display: Stub new resource objects

We want to split pipe_ctx into plane and stream resource objects.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-by: Tony Cheng <Tony.Cheng@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/inc/core_types.h | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/drivers/gpu/drm/amd/display/dc/inc/core_types.h b/drivers/gpu/drm/amd/display/dc/inc/core_types.h
index 3b3330c..6c93741 100644
--- a/drivers/gpu/drm/amd/display/dc/inc/core_types.h
+++ b/drivers/gpu/drm/amd/display/dc/inc/core_types.h
@@ -152,10 +152,21 @@ struct resource_pool {
 	const struct resource_caps *res_cap;
 };
 
+struct stream_resource {
+	int stub;
+};
+
+struct plane_resource {
+	int stub;
+};
+
 struct pipe_ctx {
 	struct dc_plane_state *plane_state;
 	struct dc_stream_state *stream;
 
+	struct plane_resource plane_res;
+	struct stream_resource stream_res;
+
 	struct mem_input *mi;
 	struct input_pixel_processor *ipp;
 	struct transform *xfm;
-- 
2.7.4

